### ISSUE TITLE
How to replace Flat UI with Flat UI Pro in Startup Framework for WordPress

### ISSUE THREAD

**Issue by iamchriswick** [designmodo/startup-support#77]
What's the best way to replace the regular Flat UI with Flat UI Pro in Startup Framework for WordPress?

**Comment by MonsterCritic** [comment]
Hi Chris,

Here is the tutorial: https://www.youtube.com/watch?v=FFVoxM16TNQ

**Comment by MonsterCritic** [comment]
Unfortunately, we do not support Startup Framework for WordPress anymore. Sorry about that.

**Comment by iamchriswick** [comment]
Is that why my Startup Framework for WordPress licence won't work either?

**Comment by MonsterCritic** [comment]
Yes. But you don't need to activate the license because there are no updates anyway.

**Comment by iamchriswick** [comment]
...well, Startup Framework for WordPress does not work at all on the latest version of  WordPress. I cant build pages, the elements won't stick to the page after they are dragged onto the layout. Any idea why?

**Comment by azariah001** [comment]
@iamchriswick  I've got this issue as well.

@MonsterCritic What was the reasoning behind dropping support for Wordpress?! Especially now that it's stopped working, like fair enough not providing updates but...... I paid good money for this and expected a decent support window. Now I'm left high and dry. I'm also a Startup Framework customer but it's nowhere near as usable without a live editor. Especially when I've just promised a client a site that incorporates a blog. And I've had zero response from support not even an "FYI this product isn't supported anymore" .... -_-

**Comment by MonsterCritic** [comment]
@azariah001 The reason was pretty obvious at that moment: to build AND support a product like this you need a team of 2-4 developers and revenue from sales was too small to cover all expenses.

@iamchriswick, @azariah001 I can offer you a free replacement from SFWP to Qards (yes, luckily it's still in development) and it's very similar.

**Comment by azariah001** [comment]
@MonsterCritic fair call ending development but zero support? Bit harsh, but anyway, business is business.

Wow. I'd seen Quards previously but never looked super close at it. It's basically the same thing...... Albeit with a heap of enhancements. I'd be happy with the trade. :grinning: If you can make that happen I would be very grateful. :smile:

Thanks for replying. :+1:

**Comment by iamchriswick** [comment]
I'm good with that! :smile:
